testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -215,6 +215,14 @@ Feature: PXE boot a Retail terminal
     Then I should see a "1 package install has been scheduled" text
     When I wait until event "Package Install/Upgrade scheduled by admin" is completed
 
+  Scenario: Chect that the PXE boot minion stays configured after reboot
+    # salt key, channels and installed packages must survive reboot
+    When I reboot the PXE boot minion
+    And I follow the left menu "Systems > Overview"
+    And I follow this "pxeboot_minion" link
+    And I wait at most 350 seconds until event "Apply states [util.syncstates, saltboot] scheduled by (none)" is completed
```

Comment:
```suggestion
When I reboot the PXE boot minion
And I wait until "salt-minion" service is active on "pxeboot_minion"
```

Reply:
If you need, refactor this step to provide a bigger timeout, and that should be good enough. For now.